pkg/sdk/go/bootstrap.go

```diff
@@ -97,7 +97,10 @@ func (sdk mfSDK) AddBootstrap(cfg BootstrapConfig, token string) (string, errors
 
 	url := fmt.Sprintf("%s/%s", sdk.bootstrapURL, configsEndpoint)
 
-	headers, _, sdkerr := sdk.processRequest(http.MethodPost, url, token, string(CTJSON), data, http.StatusOK, http.StatusCreated)
+	header := make(map[string]string)
+	header["Content-Type"] = string(CTJSON)
```

Comment:
Since this repeats for most part you can leave it at `processRequest`

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Yes, good idea - we can have a default CT, which is in the `processRequest()` function, and it is set if I do not provide any CT header. However - if I provide CT header, then this one is taken into account (not the default one).